include/deal.II/numerics/vector_tools_interpolate.h

```diff
@@ -284,6 +284,16 @@ namespace VectorTools
                       VectorType &                     vector,
                       const ComponentMask &            mask = ComponentMask());
 
+  /**
+   * Like the above function but also taking @p mapping as argument.
```

Comment:
Not related to your patch, but is the "then a default one is used" wrong in line 274 and supposed to say "then a default zero is used"?